manager/src/test/java/com/redhat/service/bridge/manager/workers/resources/ConnectorWorkerTest.java

```diff
@@ -36,9 +38,11 @@
 import static org.mockito.Mockito.when;
 
 @ExtendWith(MockitoExtension.class)
-public class ConnectorWorkerTest {
+class ConnectorWorkerTest {
```

Comment:
Beyond the scope of this PR, but we should remove all mocking from our Worker tests. I've created the following to do that https://issues.redhat.com/browse/MGDOBR-467